src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1 +1,42 @@
-INSERT INTO tab_transports(transport_direction,process,source_organ,source_compartment,target_organ,target_compartment,source_organ_type,source_compartment_type,target_organ_type,target_compartment_type) VALUES('InfluxBrainPlasmaToInterstitial','ActiveInfluxSpecificPlasmaToInterstitial','Brain','Plasma','Brain','Interstitial','ORGAN','COMPARTMENT','ORGAN','COMPARTMENT');
+INSERT INTO tab_categories(category,display_name,description,category_type,icon_name) VALUES('DiseaseStates','Disease States','Disease state parameters','Individual',NULL);
+INSERT INTO tab_container_names(container_type,container_name,display_name,description,extension,sequence,icon_name) VALUES('DISEASE_STATE','CKD','CKD Parameter container','CKD Parameter container',NULL,1,NULL);
+INSERT INTO tab_container_parameter_values(parameter_value_version,species,container_id,container_type,container_name,parameter_name,default_value,min_value,min_isallowed,max_value,max_isallowed,value_origin) VALUES('DiseaseStates','Human',5954,'DISEASE_STATE','CKD','InputGFR',60.0,0.0,1,60.0,1,92);
+INSERT INTO tab_container_parameters(container_id,container_type,container_name,parameter_name,can_be_varied,read_only,visible,can_be_varied_in_population,group_name,sequence,build_mode,building_block_type,is_input) VALUES(5954,'DISEASE_STATE','CKD','InputGFR',0,0,1,0,'BODY_ANATOMY',1,'LOCAL','INDIVIDUAL',1);
+INSERT INTO tab_container_types(container_type,display_name,description) VALUES('DISEASE_STATE','Disease State','Disease state container');
+INSERT INTO tab_containers(container_id,container_type,container_name,parent_container_id,parent_container_type,parent_container_name,visible,is_logical) VALUES(5954,'DISEASE_STATE','CKD',NULL,NULL,NULL,1,1);
+INSERT INTO tab_dimensions(dimension,kernel_unit) VALUES('Flow per surface area','l/min/1.73m²');

```

Comment:
the kernel unit should be "l/min/dm²" then

Reply:
yep correct. What is this kernel_unit used for anyways?

Reply:
I think it's not used at all. 
But it's helpful if you need to create and export an overview of parameters e.g. for a species or a population.
Such an export can be passed to an end user (without db access) for parameter value adjustments